doc/METADATA.ini

```diff
@@ -679,6 +679,43 @@ type= enum
 	RFC2822
 description= Support of some standards for a date specification.
 
+[check/permission]
+status= idea
+usedby/plugin= path
+type= enum
+description= A check to see if a certain user has correct permissions on a file. Permissions are
+	either r (read) w (write) x (execute) and any combination of them.
+	 The respective user which must have these permissions is declared in check/permission/user.
+example = "rwx ; rw ; r ; x ; w ; wx"
+
+[check/permission/user]
+status= idea
+usedby/plugin= path
+type= string
+	empty
+description= Used be check/permission to see if the given user has the correct permissions. If empty
+	it takes the current user.
+
+[check/port]
+status = idea
+usedby/plugin = network
+type = empty
```

Comment:
It is a number isn't it?

Reply:
check/port does only a check if the range is correct. A number is not required imo. type is only used to declare the type of the metavalue